react/react1/week1/todo-app/src/components/ListTodos.js

```diff
@@ -0,0 +1,11 @@
+import Todo from './Todo';
+
+export default function ListToDos({ todos }) {
+    return (
+        <ul className='todos-container'>
+            {todos.map(todo => (
+                <Todo title={todo.title} date={todo.date} />
```

Comment:
You could pass `todo={todo}` and destructure its props inside `Todo.jsx` :)

Reply:
I changed that :)

Reply:
Thank you for the review!